x-pack/plugins/observability/public/components/app/section/ux/index.tsx

```diff
@@ -72,6 +97,15 @@ export function UXSection({ bucketSize }: Props) {
       }}
       hasError={status === FETCH_STATUS.FAILURE}
     >
+      <div style={{ height: 500 }}>
+        <ExploratoryViewEmbeddable
+          attributes={seriesList}
+          reportType="data-distribution"
+          title={PAGE_LOAD_DISTRIBUTION_TITLE}
+          withActions={['save', 'explore']}
```

Comment:
Can we disable the actions for now?

<img width="1624" alt="CleanShot 2022-05-23 at 09 37 27@2x" src="https://user-images.githubusercontent.com/4104278/169767860-5522784c-44b7-40f9-9e90-762cb01360f6.png">

Reply:
Done!